src/admin/components/Fields/SinglePriceQuantityVendorSale.vue

```diff
@@ -0,0 +1,235 @@
+                <div class="commission-inner-type-middle" v-if="'combine' == commission.commission_type">
+                    <span class="commission-inner-type-middle-text">+</span>
+                </div> 
+                <div class="commission-inner-type" v-if="'flat' == commission.commission_type || 'combine' == commission.commission_type">
+                    <input :value="commission.flat" @input="updateCommissionValue( $event.target.value, 'flat', index, 'number' )" type="number" min="1" max="9999" maxlength="10" oninput="this.value=this.value.slice(0,this.maxLength||1/1);this.value=(this.value < 0) ? 0 : this.value;" class="dokan-commission-value"> 
+                    <span class="commisson-indecator" v-html="get_currency_symbol"></span>
+                </div>
+            </div>
+            <div class="dokan-new-commission-col-action">
+                <span v-if="0 != index && 0 != commission.to && '' != commission.to" @click="removeCommissionFromList(index)" class="dashicons dashicons-dismiss dokan-commission-row-delete"></span>
+            </div>
+        </div>
+    </fieldset>
+</template>
+
+<script>
+export default {
+    name : 'SinglePriceQuantityVendorSale',
+    props: [ 'allCommission', 'selectedCommissionName', 'selectedCommissionLabel' ],
+    methods: {
+        updateCommissionValue( event, field, index, type ) {
+            this.$emit(
+                'updateCommissionState',
+                {
+                    value: event,
+                    field: field,
+                    index: index,
+                    type: type
+                }
+            );
+        },
+
+        removeCommissionFromList( index, ) {
+            this.$emit(
+                'removeCommissionFromList',
+                index,
+            );
+        },
+
+        generateNextRow( input, index ) {
+            this.$emit(
+                'generateNextRow',
+                {
+                    value: input,
+                    index: index + 1,
+                }
+            );
+        },
+
+        resetRows( input, index ) {
+            this.$emit(
+                'resetRows',
+                {
+                    value: input,
+                    index: index,
+                }
+            );
+        },
+
+        async handleTOInput( e, index, to ) {
+            if ( isNaN( Number(e) ) ) {
+                this.$refs[`dokan-to${index}`].value = to;
+                return;
+            }
+
+            this.allCommission[index].from > Number(e) && '' != e ? this.showFromErrorMessage( index, this.allCommission[index].from, e ) : this.hideFromErrorMessage( index );
+            
+            await this.updateCommissionValue( e, 'to', index, 'number' );
+            await this.generateNextRow( e, index );
+        },
+
+        showFromErrorMessage( index, from, to ) {
+            this.$refs[`dokan-${this.selectedCommissionLabel}-from-msg${index}`][0].innerText = `Must be more or equal ${from}`;
```

Comment:
The text should be translatable. And the sentence will be `Must be more than or equal to `